src/pages/uxp-api/reference-js/Modules/fs/fs.md

```diff
@@ -1,29 +1,30 @@
-<a name="module-storage-filesystem" id="module-storage-filesystem"></a>
 
-# require('fs')
+<a name="fs" id="fs"></a>
 
+# require('fs')
 UXP Provides Node.js style file system API, FSAPI.
 Unlike [Entry](./uxp/Persistent%20File%20Storage/Entry/) based [File](./uxp/Persistent%20File%20Storage/File/) or [Folder](./uxp/Persistent%20File%20Storage/Folder/) classes,
 these methods can directly access a local file or folder with path or file descriptor.
 The starting point of a path in the native filesystem depends on the scheme.
 UXP supports plugin-specific storage schemes, such as "plugin:", "plugin-data:",
-and "plugin-temp:", as well as a native "file:" scheme for the path parameter.<br/>
-Note1: [UWP](https://learn.microsoft.com/en-us/windows/uwp/get-started/universal-application-platform-guide)(Universal Windows Platform)
+and "plugin-temp:", as well as a native "file:" scheme for the path parameter.<br>
+<b>Note:</b><br></br>
+1. If there are no schemes defined for the path parameter of FSAPI methods, it considers to have "file:" scheme for the path.<br></br>
+2. [UWP](https://learn.microsoft.com/en-us/windows/uwp/get-started/universal-application-platform-guide)(Universal Windows Platform)
 has the strict [File access permissions](https://learn.microsoft.com/en-us/windows/uwp/files/file-access-permissions),
-and UXP FSAPI may have access issues with anonymous filepaths.
-So, XD does not support this feature for compatibility across platforms.<br/>
```

Comment:
@kareemmyk  These should be called out in 'Known Issues' section.

Reply:
Updated in the branch ```v7.0.0-docs-review```